test/Concurrency/sendable_metatype_typecheck.swift

```diff
@@ -123,7 +123,7 @@ case q(Q.Type, Int) // expected-warning{{associated value 'q' of 'Sendable'-conf
 }
 
 struct S: Sendable {
-  var tuple: ([Q.Type], Int) // expected-warning{{stored property 'tuple' of 'Sendable'-conforming struct 'S' has non-Sendable type '([any Q.Type], Int)'}}
+  var tuple: ([Q.Type], Int) // expected-warning{{stored property 'tuple' of 'Sendable'-conforming struct 'S' has non-Sendable type 'any Q.Type'}}
```

Comment:
I like that the diagnostic structurally looks through types with conditional conformances and points out the type that is actually non-`Sendable`. It might be nice as a follow up to somehow tailor the diagnostic in the case where the property type contains the non-`Sendable` type.

Reply:
This was the inconsistency between enum elements and properties, the former printed the mismatched type and the latter the whole type. Maybe for inner type we should use `contains` or `uses` instead of `has` something like:
`stored property 'tuple' of 'Sendable'-conforming struct 'S' contains a non-Sendable type 'any Q.Type'`?

Reply:
Yeah I think `contains` sounds good!